apps/stats-web/env/.env.staging

```
@@ -1,14 +1,7 @@
-NEXT_PUBLIC_NODE_ENV=$NODE_ENV
-NEXT_PUBLIC_BASE_API_MAINNET_URL=https://console-api-mainnet-staging.akash.network
-NEXT_PUBLIC_BASE_API_SANDBOX_URL=https://console-api-sandbox-staging.akash.network
-NEXT_PUBLIC_BASE_API_TESTNET_URL=$NEXT_PUBLIC_BASE_API_MAINNET_URL
-NEXT_PUBLIC_API_BASE_URL=$NEXT_PUBLIC_BASE_API_MAINNET_URL
+VITE_NODE_ENV=$NODE_ENV
+VITE_BASE_API_MAINNET_URL=https://console-api-mainnet-staging.akash.network
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_BASE_API_MAINNET_URL key should go before the VITE_NODE_ENV key

(UnorderedKey)